src/components/article-page.js

```diff
@@ -42,24 +116,50 @@ export default class Article extends PureComponent {
 
   render() {
     const { colors, post } = this.props
+
     return (
       <ThemeProvider theme={{ colors: _.merge({}, defaultColors, colors) }}>
-        <div>
-          <LeadingBlock />
-          <Body
-            brief={_.get(post, 'brief.api_data')}
-            content={_.get(post, 'content.api_data')}
-          />
-          <Metadata
-            categories={post.categories}
-            designers={post.designers}
-            photographers={post.photographers}
-            tags={post.tags}
-            writers={post.writters}
-            engineers={post.engineers}
-            rawAutherText={post.extend_byline}
-          />
-        </div>
+        <BackgroundBlock>
+          <HeaderBlock>
+            <LeadingBlock />
+          </HeaderBlock>
+          <BodyBlock>
+            <FiveColumnsFB>
```

Comment:
I think it's hard to implement grid concepts without using CSS grid. Because without CSS grid, we cannot put the children lied cross multiple columns with simple way. That will make us need workaround to make elements horizontally centrally aligned according to the page. An alternative way is that we use variables to determine the width of each elements (like `template-columns` in CSS grid, ref: https://blog.techbridge.cc/2017/02/03/css-grid-intro/), and set alignment (horizontal position) relative to page (or body container) independently.